pandas/tests/groupby/test_transform.py

```diff
@@ -1001,3 +1002,47 @@ def test_ffill_not_in_axis(func, key, val):
     expected = df
 
     assert_frame_equal(result, expected)
+
+
+def test_transform_invalid_name_raises():
```

Comment:
Can you add GH number as comment?

Reply:
Do you want me to open a new issue or use this PR as reference?